scripts/CubitGeomConfig.cmake

```diff
@@ -0,0 +1,116 @@
+# Generated by CMake
```

Comment:
Do we need this file in the repo?

Reply:
I don't quite remember precisely, but it was part of the file I had to modify to have the plugin make to work.

I can do some test to check if this is still necessary...

Reply:
Maybe we can add this to the list of files to get diffs of for a patch.